src/gurt/tests/SConscript

```diff
@@ -23,13 +23,11 @@ def scons():
     tests = []
 
     for test in TEST_SRC:
-        flags = []
         testobj = test_env.Object(test)
         testname = os.path.splitext(test)[0]
         testprog = test_env.d_test_program(target=testname,
                                            source=testobj + gurt_targets,
-                                           LIBS=test_env["LIBS"] + ['yaml'],
-                                           LINKFLAGS=flags)
+                                           LIBS=test_env["LIBS"] + ['yaml'])
```

Comment:
What is happening here?  I assume you're removing something from LINKFLAGS but I'm not sure what?

Reply:
I also split this bit into #13201, since I'm not sure about the `-fstack-protector-strong` change I made